documentation/book/con-custom-resources-status.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-status-{context}']
+= {ProductName} custom resource status
+
+The `status` property of a {ProductName}-specific custom resource is configured to capture the _current_ state of the resource and make that information available.
+
+Status information is useful for tracking progress related to a resource achieving its _desired_ state, providing the time and reason the state of the resource changed and details of events causing delays.
+
+NOTE: The status field is different from the spec field, which describes the _desired_ state of an object.
```

Comment:
With the suggested changes I don't think we need this sentence.

Reply:
If we keep it, I suggest removing the italics.

Reply:
Removed